print-data.py

```diff
@@ -0,0 +1,348 @@
+        elif re.search(r"[)];", line) and record == True:
+            record = False
+            new_file += line
+            new_file += 'std::cout<<"awkward_slicearray_ravel_64:";index.printMe();std::cout<<":";index_.printMe();std::cout<<":"<<ndim()<<":";for (auto x : shape_)std::cout << x << ",";std::cout<<":";for (auto x : strides_)std::cout << x << ",";std::cout<<std::endl;\n'
+            num += 1
+            continue
+        new_file += line
+    x.close()
+    return new_file
+
+
+def identities():
+    new_file = """"""
+    global num
+    record = False
+    flag = 0
+    x = open(os.path.join(CURRENT_DIR, "src", "libawkward", "Identities.cpp"), "r")
+    lines = x.readlines()
+    for line in lines:
+        if re.search(r"struct\sError\serr\s=\skernel::\w+", line):
+            record = True
+        elif re.search(r"[)];", line) and record == True:
+            flag += 1
+            record = False
+            new_file += line
+            new_file += 'std::cout<<"awkward_Identities_getitem_carry_64:";printMe();std::cout<<":";carry.printMe();std::cout<<":";std::cout<<carry.length()<<":"<<width_<<":"<<length_<<std::endl;\n'
+            num += 1
+            continue
+        if flag == 2 and "return out;" in line:
+            flag = 3
+        if flag == 3 and "}" in line:
+            flag = 4
+            new_file += line
+            new_file += """ template <typename T>
+  void
+  IdentitiesOf<T>::printMe() const{
+   for (int64_t i = 0; i < length(); i++) {
+    std::cout << data()[i] << ", ";
+   }
+  }"""
+            continue
+        new_file += line
+    x.close()
+    return new_file
+
+
+def content():
+    new_file = """"""
+    global num
+    block = """"""
+    record = False
+    x = open(os.path.join(CURRENT_DIR, "src", "libawkward", "Content.cpp"), "r")
+    lines = x.readlines()
+    for line in lines:
+        if re.search(r"struct\sError\serr\s=\skernel::\w+", line):
+            block += line
+            record = True
+        elif re.search(r"[)];", line) and record == True:
+            block += line
+            stdout = get_line(block)
+            block = """"""
+            record = False
+            new_file += line
+            new_file += stdout + "\n"
+            num += 1
+            continue
+        elif record == True:
+            block += line
+        new_file += line
+    return new_file
+
+
+def index():
+    new_file = """"""
+    global num
+    record = False
+    x = open(os.path.join(CURRENT_DIR, "src", "libawkward", "Index.cpp"), "r")
+    lines = x.readlines()
+    for line in lines:
+        if re.search(r"struct\sError\serr\s=\skernel::\w+", line):
+            record = True
+        elif re.search(r"[)];", line) and record == True:
+            record = False
+            new_file += line
+            new_file += '\tstd::cout<<"awkward_Index_to_Index64:";for(int i=0;i<length();i++){std::cout <<ptr()[i]<<",";}std::cout<<":";printMe();std::cout<<":"<<length_<<std::endl;\n'
```

Comment:
@SantamRC - have you updated the PR? This line gives numerous  compilation errors:
```
/Users/yana/Projects/IRIS-Fellow/Santam/new-plan/awkward-1.0/src/libawkward/Index.cpp:294:81: error: type 'std::shared_ptr<int64_t>' (aka 'shared_ptr<long long>') does not provide a call operator
        std::cout<<"awkward_Index_to_Index64:";for(int i=0;i<length();i++){std::cout <<ptr()[i]<<",";}std::cout<<":";printMe();std::cout<<":"<<length_<<std::endl;
                                                                                       ^~~
```